iron-overlay-behavior.html

```diff
@@ -308,28 +312,25 @@
 
       this._manager.addOrRemoveOverlay(this);
 
+      // TODO handle case when it was already animating, e.g. quick open/close.
```

Comment:
lol makes sense. About this TODO in particular, it still needs more thinking hence it maybe doesn't make sense to add it at all.
e.g. if I `open()` then `close()`, what damage can do to trigger `iron-overlay-closed` if `iron-overlay-opened` wasn't triggered before it?
WDYT?